paper-toast.html

```diff
@@ -82,9 +82,8 @@
         margin: 12px;
         font-size: 14px;
         cursor: default;
-        -webkit-transition: visibility 0.3s, -webkit-transform 0.3s, opacity 0.3s;
-        transition: visibility 0.3s, transform 0.3s, opacity 0.3s;
-        visibility: hidden;
+        -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
```

Comment:
Since `paper-toast` uses `iron-overlay-behavior`, `display: none` is applied when `opened = false`. Hence it doesn't make sense to set or animate the `visibility`